dashboard/app/models/evaluation_question.rb

```diff
@@ -0,0 +1,10 @@
+class EvaluationQuestion < Multi
+  def dsl_default
+    <<ruby
+name 'Unique question name here'
+question 'Question'
+answer 'Answer1', weight, 'learning_module_name'
```

Comment:
Can we used named parameters as in https://github.com/code-dot-org/code-dot-org/pull/8295 ?